src/java/org/apache/cassandra/db/RangeTombstoneList.java

```diff
@@ -133,13 +136,21 @@ public void add(ByteBuffer start, ByteBuffer end, long markedAt, int delTime)
         if (c < 0)
         {
             addInternal(size, start, end, markedAt, delTime);
+            assertSorted();
```

Comment:
Question: should this be wrapped in a try/catch block or we are confident that this would not throw assertion error?